framework/test/jest/integration/specs/modules/chain/transactions/exceptions/signatures.spec.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('exceptions for senderPublicKey transactions', () => {
+	describe('send funds to account', () => {
+		describe('details of the accounts', () => {
+			describe('when forging block with transaction with collision publicKey', () => {
```

Comment:
I'm just highlighting this line as an example. But In general, please avoid nested `describe` blocks as much as possible. The 3 describe blocks above can be merged into one. We can always add more describe blocks when it's really needed.

If we don't do it in this PR, others will follow the same pattern and we'll end up having bunch of redundant describe blocks.

So would be great if we could improve test descriptions by following the rules below,
1 - Each `describe` block (except the root one) should start with **when**.
2 - Each `it` should start with **should**.
3 - We **don't** need describe blocks which contain only **one** test. We should add them when it's really needed.

@shuse2 there are 53 files in the PR. I think @nazarhussain mainly copied the testing structure from mocha and wasn't planning to spend more time on it. But I think these refactorings are important and maybe we can assign more people to split the files to write test cases properly.

Let me know what you think.

Reply:
@yatki I agree to all what you said and related to that I asked @shuse2 to decided on some practices e.g. nested level for describe etc and then individual person working on each tests will follow those. 

In this PR, I just extracted the skeleton from existing tests and restructured the files a bit to follow current source structure.